tests/monotouch-test/Foundation/NSDictionary2Test.cs

```diff
@@ -557,6 +557,7 @@ public void IEnumerableTest ()
 			Assert.AreEqual (2, c, "Enumerator Count");
 		}
 
+#if !MONOMAC // TODO: Messaging is internal on mac
```

Comment:
The Messaging methods should come from monotouch-test here: https://github.com/xamarin/xamarin-macios/blob/master/tests/monotouch-test/ObjCRuntime/Messaging.cs, not the ones from Xamarin.Mac.dll, so this should work just fine.